### PR TITLE
Feature/table with orders

### PR DESCRIPTION
**Pull request by screamcha**

### CODE REVIEW

File: client-app/src/components/UserPage/styles.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.table {
```
Comment (amikitevich) [inline client-app/src/components/UserPage/styles.css]: better way to write styles inside root class.
example:
```
const myComp = props => {
<div className="my-class-name"><div className="table"></div></div>
}
```

css
```
.my-container .table {
#css styles.
}
```

It encapsulates css inside component and you feel free to use .table className without the implicitly applying styles

Comment (amikitevich) [reply client-app/src/components/UserPage/styles.css]: I can tell detail about it if you need.

File: client-app/src/components/App/index.js [hunk 2]
```diff
@@ -1,17 +1,19 @@
 // @flow
 import React from 'react';
 import { Route, Link, } from 'react-router-dom';
-import Home from './components/Home';
-import SignUp from './components/SignUp';
-import SignIn from './components/SignIn';
+import Home from 'src/components/Home';
```
[the remark this answers is not in the document]
Comment (screamcha) [reply client-app/src/components/App/index.js]: It's not an absolute path, I made alias called src :)

[the remark this answers is not in the document]
Comment (amikitevich) [reply client-app/src/components/App/index.js]: Cool.

**Pull request merged**